cmd/vic-machine/create/create.go

```diff
@@ -482,10 +475,11 @@ func (c *Create) Flags() []cli.Flag {
 	debug := c.DebugFlags()
 	proxies := c.proxies.ProxyFlags(true)
 	cNetwork := c.containerNetworks.CNetworkFlags(true)
+	dns := c.dns.DNSFlags(true)
 
 	// flag arrays are declared, now combined
 	var flags []cli.Flag
-	for _, f := range [][]cli.Flag{target, compute, create, cNetwork, memory, cpu, tls, registries, proxies, syslog, iso, util, debug, help} {
+	for _, f := range [][]cli.Flag{target, compute, create, cNetwork, memory, cpu, tls, registries, proxies, dns, syslog, iso, util, debug, help} {
```

Comment:
Please place `dns` between `create` and `cNetwork` - the flags are printed in the slice's order and we should keep the same order.